module/VuFind/src/VuFind/Db/Service/UserListService.php

```diff
@@ -102,28 +105,32 @@ public function getUserListById(int $id): UserListEntityInterface
     /**
      * Get public lists.
      *
-     * @param array $includeFilter List of list ids or entities to include in result.
-     * @param array $excludeFilter List of list ids or entities to exclude from result.
+     * @param array        $includeFilter List of list ids or entities to include in result.
+     * @param array        $excludeFilter List of list ids or entities to exclude from result.
+     * @param string|array $types         Types of user lists to get. Set to an empty array to get all.
```

Comment:
For the comments, maybe `string|string[]` would be more precise... of course we have to leave it `string|array` in the actual function signature, but it never hurts to be specific in the PHPdoc.